.github/workflows/e2e-test.yml

```diff
@@ -0,0 +1,46 @@
+name: Local Env CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  local-env:
+    name: Setup Local Environment
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout Local Env repository
+      uses: actions/checkout@v4
+
+    - name: Setup and Start Local Env Dev Container
+      uses: devcontainers/ci@v0.3
+      with:
+        runCmd: |
+          kind create cluster --name to-dos --config kind-local-config.yaml --kubeconfig ./.to-dos-cluster-kubeconfig
+          helmfile cache cleanup && helmfile --environment local --namespace local -f deploy/helmfile.yaml apply
+
+    - name: Checkout UI repository
+      uses: actions/checkout@v4
+      with:
+        repository: FaceMan01/to-dos-ui
```

Comment:
Did you make any changes to UI? Can we change it to the original repo?

Reply:
No I did't make any changes. So I changed it to the original repo.